index.js

```diff
@@ -24,7 +24,8 @@ module.exports = opts => {
 	}
 
 	const title = opts.title ? `process.title = '${opts.title}';` : '';
-	const code = `${title} setInterval(() => {}, 1000 * 1000);console.log('ok');`;
+	const forceKill = opts.forceKill ? 'process.on(\'SIGTERM\', () => {console.log(\'I can only be killed by the force!!\')});' : '';
```

Comment:
You need to handle `SIGINT` too.

Reply:
Would be more readable if you moved the `process.on(\` string into a template literal variable.

Reply:
Do you mean like this:

```javascript
const forceKill = opts.onlyForceKillable ? `
		process.on('SIGTERM', () => {console.log('I can only be killed by the force!!')});
		process.on('SIGINT', () => {console.log('I can only be killed by the force!!')});
		` : '';
```

Reply:
Yes